src/media/css/site.styl

```diff
@@ -155,6 +155,34 @@ noscript {
     }
 }
 
+.error-page {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    min-height: 85vh;
+
+    > * {
+        max-width: 70%;
+    }
+}
+.error-page-heading {
+    font-weight: 100;
+    margin: 0 0 15px;
+    position: relative;
+
+    &:before {
+        background: url(../img/pretty/alert_triangle.svg) no-repeat 0 0 / 60px auto;
+        content: "";
+        height: 60px;
+        left: 50%;
+        margin-left: -30px;
```

Comment:
Both feel just as hacky but the translate will get rid of the fixed image size so will change it.